kratos/input_output/vtk_output.cpp

```diff
@@ -874,7 +995,8 @@ Parameters VtkOutput::GetDefaultParameters()
         "element_flags"                      : [],
         "condition_data_value_variables"     : [],
         "condition_flags"                    : [],
-        "gauss_point_variables"              : []
+        "gauss_point_variables"              : [],
```

Comment:
ping

Reply:
I would not change the current approach, otherwise will break current use

Reply:
I know, but if we don't update then we have
- `gauss_point_variables`
- `gauss_point_variables_in_elements`

this is super confusing, gp-values are in elements, so from a user point of view it is not clear what is what

You could easily make it backwards compatible. Not sure though if necessary, since this is a rel new feature that afaik not many ppl use

Reply:
Can we change it in a future PR?, this way we can take the time to check everywhere is used and change it (without making this PR enourmous)

Reply:
I prefer to do it now, changing the name of the setting can be easily done in python, then we have all changes together

Reply:
You won this round, but I will came back!!!!